### PR TITLE
4.0.0.pre3 release?!

### PR DESCRIPTION
**Pull request by jasonrclark**
Changes in this commit include the version bump and CHANGELOG updates.

When we're set to release, here's actually a few additional steps we'll want to do:

* [x] Run through at least a good part of the samples on all platforms
  * [x] Mac
  * [x] Windows
  * [x] Linux
* [ ] Release furoshiki 0.3.0 (shoes/furoshiki#15)
* [x] Update our gemspec to use new `furoshiki` and remove from `Gemfile`
* [ ] Merge this PR
* [ ] Release the shoooooooes!
* [ ] Since we've got new gems, update the owners on the new gems
* [ ] Send message to mailing list
* [ ] Blog post (shoes/shoesrb.com#74)
* [ ] :tada: :bell: :triumph:

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,53 @@
+4.0.0.pre3 (191 commits)
+----------------
+
+New features (1)
+----------------
+
+* In pre2 we updated our gem layout and found a few things we didn't like.
+  pre3 is now shaped how we expect to hold through 4.0!
+
+  * shoes -- Now a meta-gem with the default installation as dependencies
```
Comment (PragTob) [inline CHANGELOG]: "and no code of its own" maybe, dunno if everyone knows what a meta gem or what we use the term meta gem for :-)

**Comment by PragTob** [comment]
Seems like a good plan to me, just added running through a good part of all the samples as a point there :-) Then we shpould be set to go.

Thanks @jasonrclark !

**Comment by jasonrclark** [comment]
Testing, what's that? :P

Hoping to get some of this moved along tonight or this weekend. Shoes on!

**Comment by jasonrclark** [comment]
Will probably have seen, but updated `CHANGELOG` with your comment @PragTob, put up a PR for `furoshiki` to bump the version, updated this PR to point to that new-but-not-yet-released `furoshiki`.

Later today I'm hopefully that I'll get down to running through the samples on Mac and Windows. Will need a hand to get Linux tested.

**Comment by jasonrclark** [comment]
Oh, and with the version bump on `furoshiki` this will probably fail the packaging tests. Will get a clean re-run on it in the gap when we've released `furoshiki` and haven't quite gotten new Shoes out...

**Comment by wasnotrice** [comment]
Ran the 85 working samples on OS X. They look good.

**Comment by jasonrclark** [comment]
Updated with latest changes (since I merged #1011 which was all set to go).

Also ran through all the good samples on Windows via a locally installed gem version of `pre3`, and everything looks golden there.

Little Linux testing, a bit of writing, and I think we're good to go!

**Comment by PragTob** [comment]
I will make time for Linux testing tomorrow, promise.

test failure is due to unreleased furoshiki, so no worries.

Thanks for all the work @jasonrclark ! :)

**Comment by PragTob** [comment]
Ran through samples, noticed #1014 - otherwise it looks good :)

**Comment by jasonrclark** [comment]
With #1014 in, I'll get the final mods to the changelog and blog post together. Then I think we're ready to get this merged and cut the release. Any final blockers @PragTob @wasnotrice @KCErb?

**Comment by PragTob** [comment]
nothing from my side, do you wanna do the honors of pushing to rubygems? Otherwise I'd have time for wrapping stuff up etc. tomorrow evening :)

**Comment by jasonrclark** [comment]
Would love to do the honors. Got a little time this afternoon, so I'll get the ball rolling!

**Comment by PragTob** [comment]
superb :star:

**Comment by wasnotrice** [comment]
Great @jasonrclark! Nothing stopping us that I can see :)

**Comment by jasonrclark** [comment]
Ready to :metal: !

**Comment by KCErb** [comment]
Woohoo!! Excellent work all! :metal:

**Comment by jasonrclark** [comment]
pre3 is up on Rubygems (other steps following along)! Thanks all for the help getting this out the door. :heart: :sparkles:

**Comment by jasonrclark** [comment]
... and done. On to pre4!!!

**Comment by PragTob** [comment]
:+1: :star: 

I liked the release check list and created [a wiki page with the common stuff](https://github.com/shoes/shoes4/wiki/Release-Checklist)

**Comment by wasnotrice** [comment]
@jasonrclark thanks for the final puuuuuush!

**Comment by jasonrclark** [comment]
Great idea @PragTob! Was planning to find this issue again, but the wiki's much better going forward.

**Pull request merged**